eventstream.go

```diff
@@ -218,3 +330,86 @@ func (e *EventStream) SubscriberSpecificEvent(f func(e Event), events ...interfa
 	}
 	return &s
 }
+
+// SubscriberAllUntyped bind the eventF function to be called on all events independent on aggregate or event type
+func (e *EventStream) SubscriberAllUntyped(f func(e EventUntyped)) *subscriptionEventUntyped {
+	s := subscriptionEventUntyped{
+		eventF: f,
+	}
+	s.unsubF = func() {
+		e.lock.Lock()
+		defer e.lock.Unlock()
+
+		for i, sub := range e.allEventsUntyped {
+			if &s == sub {
+				e.allEventsUntyped = append(e.allEventsUntyped[:i], e.allEventsUntyped[i+1:]...)
+				break
+			}
+		}
+	}
+	s.subF = func() {
+		e.lock.Lock()
+		defer e.lock.Unlock()
+
+		e.allEventsUntyped = append(e.allEventsUntyped, &s)
+	}
+	return &s
+}
+
+// SubscriberAggregateTypeUntyped bind the eventF function to be called on events on the aggregate type
+func (e *EventStream) SubscriberAggregateTypeUntyped(f func(e EventUntyped), aggregateTypes ...string) *subscriptionEventUntyped {
```

Comment:
Is there perhaps a better name for this method? The part `...TypeUntyped` makes it confusing?